users/utils.py

```diff
@@ -0,0 +1,26 @@
+import json, jwt
+
+from django.conf  import settings
+from django.http  import JsonResponse
+
+from users.models import User
+
+def signin_decorator(func):
+
+    def wrapper(self, request, *args, **kwargs):
+
+        try:
+            token        = request.headers.get('Authorization', None)
+            payload      = jwt.decode(token, settings.SECRET_KEY, settings.ALGORITHM)
+            user         = User.objects.get(id=payload['id'])
+            request.user = user
+
+            return func(self, request, *args, **kwargs)
+
+        except jwt.exceptions.DecodeError:
+            return JsonResponse({'message':'INVALID_TOKEN'}, status=401)
+
+        except User.DoesNotExist:
+            return JsonResponse({'message':'INVALID_USER'}, status=401)
+            
+    return wrapper
```

Comment:
너무 많은 줄바꿈이 있는것 같습니다!! 꼭 필요하지 않고 목적에 따라 나누는게 아니라면 굳이 해줄 필요가 없어 보입니다
```suggestion
def signin_decorator(func):
    def wrapper(self, request, *args, **kwargs):
        try:
            token        = request.headers.get('Authorization', None)
            payload      = jwt.decode(token, settings.SECRET_KEY, settings.ALGORITHM)
            user         = User.objects.get(id=payload['id'])
            request.user = user

            return func(self, request, *args, **kwargs)

        except jwt.exceptions.DecodeError:
            return JsonResponse({'message':'INVALID_TOKEN'}, status=401)
        except User.DoesNotExist:
            return JsonResponse({'message':'INVALID_USER'}, status=401)
    return wrapper
```